### DISCUSSION TITLE
Deprecated StorageType.persistent

### DISCUSSION THREAD

**Discussion by hckhanh** [fingerprintjs/fingerprintjs#842] [General]
I saw this warning on Chrome, I am using Fingerprint Pro

```
StorageType.persistent is deprecated. Please use standardized navigator.storage instead.
```

A trace leads to this url:

```
https://fpnpmcdn.net/v3/<a string like id>/loader_v3.8.1.js
```

**Comment by makma** [comment]
Hello @hckhanh,
I tried to reproduce it on my local machine on Chrome with no luck.
Could you please share more detailed information on how to reproduce and possibly also a page with a warning in the console? Please, also share with us your operating system, Chrome version, information about possible browser extensions, and incognito window usage.
Thank you.

**Comment by hckhanh** [comment]
Hi @makma, I will give you more info:

![image](https://user-images.githubusercontent.com/6380436/204821518-8781f403-505d-4124-b978-e33b53efa519.png)

Operating system: Windows 11
Chrome version: Version 107.0.5304.122 (Official Build) (64-bit)

Do you need anything else?

**Reply by makma** [reply]
Hello,
I've just tested exactly the same browser version on win 11 and I couldn't reproduce the issue.

Do you use any browser extensions?
Can you reproduce the same behavior with disabled extensions?
Can you reproduce the behavior in the incognito window?
Can you provide us with a live URL where we can try to reproduce the issue?

Thanks

**Reply by makma** [reply]
@hckhanh, Please disregard my previous questions. One of our engineers successfully reproduced the issue. We will fix it in the next release.
Thank you.

**Comment by GalacticHypernova** [comment]
Just wanted to add to this that this is still happening, I am getting this warning as well

**Reply by Finesse** [reply]
The fix was released on December 29th 2022. What Agent version do you use?

**Reply by GalacticHypernova** [reply]
I'm sorry for the late response. I was away from home the last couple of days. How do I check that exactly?

**Reply by Finesse** [reply]
If use use NPM or Yarn to install FingerprintPro, check the `@fingerprintjs/fingerprintjs-pro` version in `package-lock.json` / `yarn.lock`. If you use a `<script>` tag or an `import` call, open the script URL and see the version on the top of the file.

**Reply by GalacticHypernova** [reply]
Oh, I'm using free version and it says @fingerprintjs/fingerprintjs@^3.3.0

**Reply by Finesse** [reply]
@GalacticHypernova I can't reproduce it with latest version of `@fingerprintjs/fingerprintjs` in Chrome 111. Please try to update.

**Reply by GalacticHypernova** [reply]
Alright, I'll try, thank you

**Reply by GalacticHypernova** [reply]
Update: updated to 3.4.0 and it fixed the issue. Thanks for the help!
